.changes/v2.17.0/503-enhancement.md

```diff
@@ -0,0 +1,3 @@
+* Added new field `Hypervisor` in `QueryResultVMRecordType` struct:
+    * String field containing the hostName value from the XML VMAdminRecord body retrieved
```

Comment:
@thomsmoreau This comment is about the title of the PR (I don't have a better place to put it):

Please expand the PR title to explain what has been done. E.g.:
"Add hostName field to VM record type"